skin/src/main/java/org/researchstack/skin/ui/SplashActivity.java

```diff
@@ -30,14 +30,14 @@ public void onDataReady()
                 .initialize(this)
                 .compose(ObservableUtils.applyDefault())
                 .subscribe(response -> {
+                    boolean boarded = AppPrefs.getInstance(this).isOnboardingComplete();
+                    boolean signedin = DataProvider.getInstance().isSignedIn(this);
+                    boolean emailverified = AppPrefs.getInstance(this).isEmailVerified();
 
-                    if(AppPrefs.getInstance(this).isOnboardingComplete() ||
```

Comment:
This is a significant behavior change. Looking at the documentation for setOnboardingComplete in AppPref, this behavior would be unexpected.

Also, I see that OnboardingActivity already calls EmailVerificationActivity. So it would make sense to me the OnboardComplete flag in AppPrefs could handle it.